drape_frontend/stylist.cpp

```diff
@@ -13,6 +13,11 @@
 
 namespace df
 {
+  // Minimum possible log2() value for the float (== -126).
+  double const minLog2 = std::log2(std::numeric_limits<float>::min());
```

Comment:
nit: kMinLog2 and you can put this constants under anonimous namespace below:
namespace df
{
namespace
{
double const kMinLog2 = ...
double constexpr kMinLinesDepth = ...

Reply:
Done!